services/gcp/iap/tunnelDestGroups.yml

```diff
@@ -0,0 +1,50 @@
+name: Identity Aware Proxy tunnel destination groups resource type.
+description: >-
+  Refers to a particular group of VMs in you project.  VMs are collected in a list of either CIDRs or FQDNs.
+scope: HIGH
+notes: >-
+  IAP is used to control access to cloud services.  Changes to IAP related settings could remove access from mission-critical applications or 
+  grant an attacker access to sensitive resources.
+privileges:
+  accessViaIAP:
+    risks: [impact:dos, escalation:network]
+    notes: >-
+      Gives a principal access to a particular IAP secured tunnel destination group.  This could create a dos if access is removed.  An attacker could also use this to gain access to a new
+    links:
+      - https://cloud.google.com/iap/docs/managing-access
+  create:
+    risks: [escalation:network]
+    notes: >-
+      Creates a new destination group.  An attacker could exploit this to gain access to new machines.
+  delete:
+    risks: [impact:dos, destruction:network]
+    notes: >-
+      Deletes an existing tunnel destination group.  This could create a dos if the deleted item is used by other services since they would no longer have access.
+  get:
+    risks: [discovery:network, discovery:infra]
+    notes: >-
+      Retrieves an existing tunnel destination group.
+  getIamPolicy:
+    risks: [discovery:policy, discovery:account]
+  list:
+    risks: [discovery:network, discovery:infra]
```

Comment:
Calling this will show the user a list of CIDRs and domain names.  I'm not sure if that counts as infra.